runtime/vara/src/governance/tracks.rs

```diff
@@ -259,6 +307,15 @@ impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
                 frame_system::RawOrigin::Root => Ok(0),
                 _ => Err(()),
             }
+        } else if let Ok(frame_system::RawOrigin::Signed(signer)) =
```

Comment:
Should we add the similar condition for the pauser, shouldn't we?

Reply:
thanks